pets/cassandra/on-start.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -e
+
+# Parse out hostname, formatted as: petset_name-index
+IFS='-' read -ra ADDR <<< "$(hostname)"
+MY_ID=$(expr "1" + "${ADDR[1]}")
+CFG=/etc/cassandra/cassandra.yaml
```

Comment:
We might want the cassandra.yaml that is in the repo ....  Then it is copied in. There are too many configuration in it to use the stock version, and then have some bash fu tweak them.  

This probably can be addressed in a future PR.